src/components/orders/OrderSurplusDisplay/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { Order } from 'api/operator'
+
+import { formatSmart, safeTokenName } from 'utils'
+
+const Wrapper = styled.div`
+  display: flex;
+
+  & > * {
+    margin-right: 0.25rem;
+  }
+
+  & > :last-child {
+    margin-right: 0;
+  }
+`
+
+const Surplus = styled.span`
+  ::before {
+    content: '+';
+  }
+  ::after {
+    content: '%';
+  }
```

Comment:
why is this better than doing `+{ formatSmart({... }) }%` in line 54

Reply:
Just because I know how to do that with CSS now :smile: 
But I also thought it looks cleaner
Would you rather have it like you suggest? I don't really mind.

Reply:
its more declarative pt 2 but this is also good cuz CSS skillz +1

Reply:
In general my approach with pseudo-elements is: Is it a layout element or actual content? In this case I think the % sign is part of the content and would make it part of the 'normal' text. Mainly for accessibility purposes too.

E.g. IMO would be different if we use it to add a '-' separator which I'd regard as a visual element.

Reply:
Makes sense.
Will change in another PR.

Reply:
https://github.com/gnosis/gp-ui/pull/203